arches/app/etl_modules/base_import_module.py

```diff
@@ -46,6 +46,9 @@ def __init__(
         if self.request:
             self.userid = request.user.id
             self.moduleid = request.POST.get("module")
+            self.loadid = request.POST.get("load_id")
+            if loadid is not None and self.loadid != loadid:
+                raise ValueError("loadid from request does not match loadid argument")
```

Comment:
This is not necessary for the fix, I just noticed this potential for drift and added a sanity check in a separate commit.

Reply:
This seems to be a nice check.